docs/src/pages/components/card/demo.tsx

```diff
@@ -59,7 +59,7 @@ export const CardDemo = () => {
         backgroundColor={tokens.colors.background.secondary}
         padding={tokens.space.medium}
       >
-        <Card {...props}>I'm a card!</Card>
+        <Card {...props}>{`I'm a card!`}</Card>
```

Comment:
What does this fix? Was Next adding extra `p` tags?

Reply:
`next lint` was warning about `'` being an unescaped entity:
> https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md

I've never really had an issue personally...